### PR TITLE
Task/stw 30

### PR DESCRIPTION
**Pull request by ilya-shaydullin**
Add the onecall API method. See for more info https://openweathermap.org/api/one-call-api

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -19,23 +19,21 @@
...
 sourceCompatibility = 1.8
 
-group = "net.aksingh"
+group = "com.icerockdev"
 archivesBaseName = "owm-japis"
-version = "2.5.3.0"
+version = "2.6.3.0"
 
 repositories {
+  mavenLocal()
```
Comment (Alex009) [inline build.gradle]: some dependencies pulled from local?

File: build.gradle [hunk 2]
```diff
@@ -46,97 +44,13 @@ dependencies {
   compile "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlin_version"
 }
 
-task javadocJar(type: Jar) {
-  classifier = "javadoc"
-  from javadoc
-}
```
Comment (Alex009) [inline build.gradle]: why it's removed too?

**Review by Alex009 — COMMENTED** [review]
can we also send pull request to original repo? without changes of build.gradle

File: build.gradle (hunk 2 above)
Comment (ilya-shaydullin) [reply build.gradle]: Fix

File: build.gradle (hunk 1 above)
Comment (ilya-shaydullin) [reply build.gradle]: Fix

**Review by Alex009 — APPROVED** [review]

**Pull request merged**